db_builder/Source/main.cpp

```diff
@@ -24,9 +28,22 @@ Json::Value getJsonFromFile(const std::string& filename, const std::string& down
     std::ifstream file(filename);
     if (is_empty(file))
     {
+        file.close();
+        
+        fd::FileDownloader downloader;                              
+        downloader.createDirectory(directoryPath_);
+    
+        std::string links_[] = {download_url};
+        std::string fileNames_[] = {filename};
+        downloader.addLinks(links_, fileNames_, (sizeof(links_)/sizeof(links_[0])));
+        downloader.download(false);
+        
+        std::string filename_ = directoryPath_ + "/" + filename;
+        file.open(filename_);
+        
         // ya ya. This sucks, but it is sufficient for this PoC
-        throw std::runtime_error(
-            std::string("There are no json in " + filename + " Download it from " + std::string(download_url)));
+        //throw std::runtime_error(
```

Comment:
czemu to zakomentowales ?

Reply:
to był dawny komunikat Szymona, który mówił o braku jsonów. Teraz zamiast pokazywania się tego komunikatu, pobierany jest brakujący plik. Ten fragment nie jest już raczej potrzebny, usunę go (wcześniej zapomniałem)